generators/heroku/templates/heroku.gradle.ejs

```diff
@@ -38,4 +56,17 @@ gradle.taskGraph.whenReady {taskGraph ->
         }
     }
 }
-<%_ } _%>
+<%_ } else { _%>
+configurations {
+    all {
+        resolutionStrategy {
+            // force dependency version as used bonsai add-on as of now only supports 7.10.x
+            // https://github.com/jhipster/generator-jhipster/issues/18650
+            force 'org.elasticsearch.client:elasticsearch-rest-client:7.10.2'
+            force 'org.elasticsearch.client:elasticsearch-rest-high-level-client:7.10.2'
```

Comment:
versions in gradle-property?

Reply:
Would like to keep it enclosed in the heroku specific file/config, will try to define a property there not in gradle.properties